lib/release_manager/cli/release_mod_cli.rb

```diff
@@ -38,6 +38,9 @@ def self.run
         opts.on('-a', '--auto', 'Run this script without interaction') do |c|
           options[:auto] = c
         end
+        opts.on('-l', '--level', 'Semantic versioning level to bump (major.minor.patch), defaults to patch') do |c|
+          options[:level] = c
+        end
         opts.on('-m', '--module-path [MODULEPATH]', "The path to the module, defaults to #{Dir.getwd}") do |c|
           options[:path] = c
         end
```

Comment:
Put this here: `options[:level] ||= 'patch'` after the opts.on

Reply:
In order to store the option you have use the following syntax.  `opts.on('-l', '--level [LEVEL]', 'blah blah'`

Also in your description please separate the levels with commas `( major,minor,patch)`

Additionally, you should also supply some validation  %w(major minor patch).include?(c)